app/scripts/modules/titus/securityGroup/securityGroupPicker.component.ts

```diff
@@ -112,10 +112,10 @@ class SecurityGroupPickerController implements ng.IComponentController {
       }
     }
     this.availableGroups = newRegionalSecurityGroups.sort((a, b) => {
-      if (this.groupsToEdit.includes(a.id)) {
+      if (this.groupsToEdit && this.groupsToEdit.includes(a.id)) {
         return -1;
       }
-      if (this.groupsToEdit.includes(b.id)) {
+      if (this.groupsToEdit && this.groupsToEdit.includes(b.id)) {
```

Comment:
I started down this route but ended up with https://github.com/spinnaker/deck/pull/2944